tests/components/ring/test_binary_sensor.py

```diff
@@ -1,87 +1,22 @@
 """The tests for the Ring binary sensor platform."""
-from asyncio import run_coroutine_threadsafe
-import unittest
 from unittest.mock import patch
 
-import requests_mock
+from .common import setup_platform
 
-from homeassistant.components import ring as base_ring
-from homeassistant.components.ring import binary_sensor as ring
 
-from tests.common import get_test_home_assistant, load_fixture, mock_storage
-from tests.components.ring.test_init import ATTRIBUTION, VALID_CONFIG
+async def test_binary_sensor(hass, requests_mock):
+    """Test the Ring binary sensors."""
+    with patch(
+        "ring_doorbell.Ring.active_alerts",
+        return_value=[{"kind": "motion", "doorbot_id": 987654}],
+    ):
+        await setup_platform(hass, "binary_sensor")
 
+    motion_state = hass.states.get("binary_sensor.front_door_motion")
+    assert motion_state is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4821145)